### DISCUSSION TITLE
Copy paste cell to multiple cell selection?

### DISCUSSION THREAD

**Discussion by jonathanmotif** [AntlerVC/firetable#877] [Support / Q&A] [answered]
Can I paste a copied cell to a selection of cells like you are able to do in a spreadsheet?

ie. drag a cell across a selection or pasting across a selection?

I have a lot of data that's "Yes, No" across hundreds of rows. 

This seems like a fairly basic operation but I cannot find it.

**Comment by harinij** [comment, marked as answer]
Hi @jonathanmotif currently we support right click copy and paste for text or compatible field types.

We are working on making the table more accessible but currently drag type of copy paste action is not supported.
